org.daidouji.oneko.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.daidouji.oneko",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/bin/oneko",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=x11",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "install desktop",
```

Comment:
this should be part if the "main" module below. there is not point splitting it.